path-traversal/COP4331-Large-Project_FoodGram/api.js

```
@@ -0,0 +1,611 @@
+      Please click the following link to reset your password.
+      https://foodgram-demo.herokuapp.com/reset-password?token=${user._id}
+  `
+  };
+  try
+  {
+      sgMail.send(msg);
+      //req.flash('Success', 'Thanks for registering!');
+      console.log('Email sent')
+      //res.redirect('/');
+      error = 'Email sent correctly';
+      var ret = {id: 1, error: 'An email was sent to your inbox!'}
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ret'. Consider to delete it or rename it.

'ret' is defined here:


(lint/suspicious/noRedeclare)